lib/Doctrine/Common/Cache/CacheProvider.php

```diff
@@ -135,6 +136,19 @@ public function save($id, $data, $lifeTime = 0)
     /**
      * {@inheritdoc}
      */
+    public function deleteMultiple(array $keys)
+    {
+        $namespacedKeys = array();
+        foreach ($keys as $key) {
+            $namespacedKeys[] = $this->getNamespacedId($key);
```

Comment:
the `array_values` is not important if `doDeleteMultiple` doesn't care about the array keys.